tests/core/scenario/test_scenario_manager.py

```diff
@@ -277,6 +284,8 @@ def test_scenario_manager_only_creates_data_node_once():
 
 
 def test_notification_subscribe(mocker):
+    _Scheduler._set_job_config(Config.configure_job_executions(mode=JobConfig._DEVELOPMENT_MODE))
```

Comment:
Maybe can you put this config in the conftest and specify the standard config only where needed?